src/chrome/content/rules/Suddeutsche_Zeitung.xml

```diff
@@ -153,7 +147,6 @@ Fetch error: http://www.sueddeutsche.de/favicon.ico => https://www.sueddeutsche.
 	<test url="http://geoservice.sueddeutsche.de/" />
 	<test url="http://global.sueddeutsche.de/assets/img/header/sprite_header.png" />
 	<test url="http://id.sueddeutsche.de/" />
-	<test url="http://jetzt.sueddeutsche.de/" />
```

Comment:
Please move it to the `Nonfunctional` or `Problematic` domains section.

Reply:
it is gone. The youth magazine lives on www.jetzt.de

Reply:
The old host is still alive from here, but times out on https.